content/build.ts

```diff
@@ -0,0 +1,500 @@
+ * Creates build configuration using current directory.
+ * Determines root and output directories relative to this file.
+ *
+ * @returns Build configuration
+ */
+export const createBuildConfig = (): BuildConfig => {
+  return {
+    rootDir: __dirname,
+    outputDir: path.join(__dirname, "lib"),
+  };
+};
+
+/**
+ * Logs build results to console.
+ * Displays statistics for all 8 content types with checkmarks.
+ *
+ * @param result - Build statistics
+ */
+export const logBuildResults = (result: BuildResult): void => {
+  console.log(`✓ Built industry.json with ${result.industriesCount} industries`);
+  console.log(`✓ Built tasks.json with ${result.tasksCount} tasks`);
+  console.log(`✓ Built actions.json with ${result.actionsCount} anytime actions`);
+  console.log(`✓ Built certifications.json with ${result.certificationsCount} certifications`);
+  console.log(`✓ Built filings.json with ${result.filingsCount} filings`);
+  console.log(`✓ Built fundings.json with ${result.fundingsCount} fundings`);
+  console.log(
+    `✓ Built license-calendar-events.json with ${result.licenseCalendarEventsCount} license-related calendar events`,
+  );
+  console.log(
+    `✓ Built license-reinstatements.json with ${result.licenseReinstatementsCount} license reinstatements`,
+  );
+};
+
+// ============================================================================
+// ENTRY POINT - Composition Root
+// ============================================================================
+
+/**
+ * Main build execution.
+ * Composes all dependencies and executes build process.
+ *
+ * This script generates static JSON files for all content types:
+ * - Industries: Business industry configurations with roadmap settings
+ * - Tasks: All task types from multiple directories (tasks, license-tasks, etc.)
+ * - Actions: Anytime Action tasks
+ * - Certifications: Business certifications
+ * - Filings: Tax filings
+ * - Fundings: Funding opportunities
+ * - License Calendar Events: License-related calendar events
+ * - License Reinstatements: License reinstatement actions
+ *
+ * Each content type generates two files:
+ * - {type}.json - Minified for production use
+ * - {type}.pretty.json - Pretty-printed for development/debugging
+ *
+ * Content parsing is delegated to shared loader functions which:
+ * - Apply proper markdown converters specific to each content type
+ * - Handle multiple source directories where applicable
+ * - Parse frontmatter and markdown content
+ * - Link related content (e.g., task dependencies)
+ */
+export const main = (): void => {
+  // Create ports (dependency injection)
+  const fileSystem = createFileSystemPort();
+
+  // Create configuration
+  const config = createBuildConfig();
+
+  // Execute build (content parsing handled by shared loaders)
+  const result = executeBuild(fileSystem, config);
+
+  // Log results
+  logBuildResults(result);
+};
+
+// Execute build only when run directly (not when imported)
+/* istanbul ignore next */
```

Comment:
It's the only line that can't be checked, because if you're checking its failure, then your runtime is broken.